web/src/upload_widget.ts

```diff
@@ -23,6 +67,173 @@ function is_image_format(file: File): boolean {
     return supported_types.includes(type);
 }
 
+function hide_elements(selectors: string[]): (HTMLElement | SVGElement)[] {
+    const hidden_elements: (HTMLElement | SVGElement)[] = [];
+    const interval_id = setInterval(() => {
+        let all_elements_found = true;
+        for (const selector of selectors) {
+            const elements = document.querySelectorAll(selector);
+            if (elements.length > 0) {
+                for (const element of elements) {
+                    if (element instanceof HTMLElement || element instanceof SVGElement) {
+                        element.style.visibility = "hidden";
+                        hidden_elements.push(element);
+                    }
+                }
+            } else {
+                all_elements_found = false;
+            }
+        }
+        if (all_elements_found) {
+            // Stop checking once all elements are found and hidden
+            clearInterval(interval_id);
+        }
+    }, 2); // Check every 2 milliseconds
+    return hidden_elements;
+}
+
+function display_dashboard_after_timeout(is_build_widget: boolean): void {
+    const dashboard_element = document.querySelector(".uppy-Dashboard-inner");
+    if (dashboard_element && dashboard_element instanceof HTMLElement) {
```

Comment:
Prefer using jquery -`$(".uppy-Dashboard-inner")` over `document.` for selecting elements

Reply:
Wherever it's possible we'll change to jquery.